lib/services/API/country_educational_institute_api.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:mobile_app/constants.dart';
+import 'package:mobile_app/models/failure_model.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+class CountryInstitueAPI {
```

Comment:
Typo

Reply:
Also can you make a contract by defining a abstract class and implementing it.

Reply:
Sure I will try that.